nixos/tests/ksm.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test-python.nix ({ pkgs, ...} :
+
+{
+  name = "ksm";
+  meta = with pkgs.lib.maintainers; {
```

Comment:
```suggestion
import ./make-test-python.nix ({ lib, ...} :

{
  name = "ksm";
  meta = with lib.maintainers; {
```

Reply:
Thanks, I never noticed it (pasted this from an old template).